### PR TITLE
add patch, patches

### PR DESCRIPTION
**Pull request by rawlingsj**
Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

**Review by cpanato — APPROVED** [review]

**Review by imjasonh — APPROVED** [review]

File: patch.yaml [hunk 1]
```diff
@@ -13,6 +13,20 @@ package:
   dependencies:
     runtime:
 
+secfixes:
+  2.7.6-r7:
```
Comment (kaniini) [inline patch.yaml:17]: `epoch` needs to be bumped to make the version `2.7.6-r3`.

These patches should be acknowledged using the Wolfi version, not the Alpine version.  So, all of these should be `2.7.6-r3`.

File: patch.yaml [hunk 2]
```diff
@@ -30,6 +44,77 @@ pipeline:
     with:
       uri: https://ftp.gnu.org/gnu/patch/patch-${{package.version}}.tar.gz
       expected-sha256: 8cf86e00ad3aaa6d26aca30640e86b0e3e1f395ed99f189b06d4c9f74bc58a4e
+
+  - uses: fetch
```
Comment (kaniini) [inline patch.yaml:48]: Please copy the patches into our repository, instead of fetching them from Alpine's cgit instance.

**Review by kaniini — CHANGES_REQUESTED** [review]
`epoch` needs to be bumped, `secfixes` needs to reflect the wolfi version introducing the fix, patches need to be copied instead of fetched.

File: patch.yaml (hunk 2 above)
Comment (rawlingsj) [reply patch.yaml]: thanks - done

File: .github/workflows/push-production.yaml [hunk 3]
```diff
@@ -182,6 +182,8 @@ jobs:
       - uses: chainguard-dev/actions/inky-build-pkg@main
         with:
           package-name: patch
+          empty-workspace: false
```
Comment (kaniini) [inline .github/workflows/push-production.yaml:185]: This isn't needed anymore, the action now just creates an empty directory if one didn't already exist.

**Review by kaniini — CHANGES_REQUESTED** [review]

**Review by kaniini — APPROVED** [review]

**Pull request merged**